overrides/scripts/altplates.zs

```diff
@@ -0,0 +1,6 @@
+//Alternative Induction Smelter Recipes for Plates
+mods.thermalexpansion.Smelter.addRecipe(500, <mw:AluminumIngot> * 2, <minecraft:stone_pressure_plate>, <mw:AluminumPlate>);
+mods.thermalexpansion.Smelter.addRecipe(750, <minecraft:iron_ingot> * 2, <minecraft:stone_pressure_plate>, <roadstuff:itemIronPart:1>);
+mods.thermalexpansion.Smelter.addRecipe(1000, <mw:SteelIngot> * 2, <minecraft:stone_pressure_plate>, <mw:MiniSteelPlate>);
+mods.thermalexpansion.Smelter.addRecipe(1250, <mw:MiniSteelPlate> * 2, <mw:SteelIngot>, <mw:SteelPlate>);
+mods.thermalexpansion.Smelter.addRecipe(1500, <mw:SteelPlate>, <mw:MiniSteelPlate> * 2, <mw:BigSteelPlate>);
```

Comment:
Not sure if using pressure plates here is the best idea if they're not needed for the "normal" recipe

Reply:
Normally I would have used one of the respective ingots for each input, but this would fail with automation. I thought that the Stone Presureplate 1. doesn't need a lot of resources and 2. you can say that you can use the Pressure Plate to shape the other plates